apps/gitness/src/pages-v2/repo/repo-layout.tsx

```diff
@@ -6,7 +6,7 @@ import { useTranslationStore } from '../../i18n/stores/i18n-store'
 const RepoLayout = () => {
   return (
     <>
-      <div className="fixed top-0 z-30 ml-56 w-full bg-background-1">
+      <div className="breadcrumbs">
```

Comment:
This needs to be checked why classes are not getting added

Reply:
It used to apply previously, probably there's some change in Tailwind config or else there's some build issue.